includes/admin/upgrades/upgrade-functions.php

```diff
@@ -3815,3 +3819,15 @@ function give_v270_store_stripe_account_for_donation_callback() {
 		give_set_upgrade_complete( 'v270_store_stripe_account_for_donation' );
 	}
 }
+
+/**
+ * Removes any leftover export files that should've been deleted
+ *
+ * @since 2.9.0
+ */
+function give_v290_remove_old_export_files() {
+	@unlink( WP_CONTENT_DIR . '/uploads/give-payments.csv' );
+	@unlink( WP_CONTENT_DIR . '/uploads/give-donors.csv' );
+
+	give_set_upgrade_complete( 'v290_remove_old_export_files' );
```

Comment:
@JasonTheAdams For automatically upgrade you do not have to store id in the database if not any logic depends upon it. I will remove it.

Reply:
fixed: https://github.com/impress-org/givewp/compare/e96b9571c95b...a3c608f60b0c